src/ledger.js

```diff
@@ -40,7 +40,7 @@ import {
   DirectKeystoreInteraction,
 } from "./interaction";
 
-import IMAGES from "./images";
+//import IMAGES from "./images";
```

Comment:
Wow, we're not using the images at all anymore? I guess we don't use them or have them yet for signing?

Reply:
we were only showing images for xpubs/pubkeys at the moment

Reply:
> we were only showing images for xpubs/pubkeys at the moment

See also https://github.com/unchained-capital/caravan/issues/112